src/tools/tidy/src/ui_tests.rs

```diff
@@ -7,7 +7,7 @@ use std::path::Path;
 
 const ENTRY_LIMIT: usize = 1000;
 // FIXME: The following limits should be reduced eventually.
-const ROOT_ENTRY_LIMIT: usize = 1418;
+const ROOT_ENTRY_LIMIT: usize = 1402;
```

Comment:
This equals the current entry number, right (I'm seeing the number here: https://github.com/jyn514/rust/tree/group-rustdoc-tests/src/test/ui)? If so, we may encounter tidy error when merging due to other PRs that add a test to the root. So 1405~1410 would be better to avoid that failure.

Reply:
> If so, we may encounter tidy error when merging due to other PRs that add a test to the root. 

Isn't that the point, so that the number of tests in the root doesn't grow? CI should have failed when they were originally opened, since it would have brought the number from 1418 to 1419.

Reply:
I originally allowed a little leeway to the limit value on #79959 to avoid failures on merging and not to get contributors confused too frequently. And we also allow leeway when moving tests, e.g., https://github.com/rust-lang/rust/pull/81387#issuecomment-770074719. But now, someone adds some tests and the number of entries seems to have almost reached the limit. So, in order to allow a little leeway again, I'd like to increase the limit a bit here. Even if the limit is not too strict, I think it makes sense to prevent adding too many tests on the root.

Reply:
This PR doesn't change the leeway compared to before. There were 1418 tests in src/test/ui before, and there are 1402 now. So it still doesn't allow adding new tests, but that's not a new change.

I'm ok with making this change, I'm just kind of confused why you want me to make it specifically in this PR.

Reply:
> This PR doesn't change the leeway compared to before. There were 1418 tests in src/test/ui before, and there are 1402 now. So it still doesn't allow adding new tests, but that's not a new change.

I answered above, `in order to allow a little leeway again`.

> I'm just kind of confused why you want me to make it specifically in this PR.

Because this touches the limit. It's easier to change it while we're at it :)

Reply:
Ok, done.